functions/EmuScripts/emuDeckRetroArch.sh

```diff
@@ -1541,6 +1543,35 @@ RetroArch_melonDS_setConfig(){
 	RetroArch_setOverride 'nds.cfg' 'melonDS'  'rewind_enable' '"false"'
 }
 
+RetroArch_melonDSDS_setUpCoreOpt(){
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS'  'melonds_audio_bitrate' '"Automatic"'
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS'  'melonds_audio_interpolation' '"None"'
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS'  'melonds_boot_directly' '"enabled"'
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS' 'melonds_console_mode' '"DS"'
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS'  'melonds_dsi_sdcard' '"disabled"'
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS'  'melonds_hybrid_ratio' '"2"'
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS'  'melonds_hybrid_small_screen' '"Duplicate"'
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS'  'melonds_jit_block_size' '"32"'
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS'  'melonds_jit_branch_optimisations' '"enabled"'
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS'  'melonds_jit_enable' '"enabled"'
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS'  'melonds_jit_fast_memory' '"enabled"'
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS'  'melonds_jit_literal_optimisations' '"enabled"'
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS'  'melonds_opengl_better_polygons' '"enabled"'
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS'  'melonds_opengl_filtering' '"nearest"'
+	RetroArch_setOverride 'melonDS DS.opt' 'melonDS DS'  'melonds_opengl_renderer' '"enabled"'
```

Comment:
Equivalent to `melonds_render_mode = "opengl"`, but the `software` renderer is actually better in most cases. The software renderer is faster and more accurate, and I recommend that it be used as the default. The OpenGL renderer's chief advantage is its ability to upscale 3D graphics, although it doesn't do so correctly in scenes that use 3D on both screens.